infra/azure.tf

```diff
@@ -102,19 +102,41 @@ cat <<'CRON' > /root/daily-reset.sh
 set -euo pipefail
 echo "$(date -Is -u) start"
 
-scale_sets='${jsonencode(concat(local.ubuntu.azure, local.windows.azure))}'
+AZURE_PAT=${secret_resource.vsts-token.value}
+
+scale_sets='$(az vmss list | jq -c '[.[] | {name, size: .sku.capacity}]')'
```

Comment:
We're now getting this directly from the "current state" on Azure, rather than from the Terraform config.